diff --git a3-client/src/introsde/assignment/soap/HealthMeasureHistory.java a3-client/src/introsde/assignment/soap/HealthMeasureHistory.java
index c82e04f..74cc83e 100644
--- a3-client/src/introsde/assignment/soap/HealthMeasureHistory.java
+++ a3-client/src/introsde/assignment/soap/HealthMeasureHistory.java
@@ -4,6 +4,7 @@ package introsde.assignment.soap;
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlSchemaType;
 import javax.xml.bind.annotation.XmlType;
 import javax.xml.datatype.XMLGregorianCalendar;
@@ -38,6 +39,7 @@ import javax.xml.datatype.XMLGregorianCalendar;
     "timestamp",
     "value"
 })
+@XmlRootElement
 public class HealthMeasureHistory {
 
     protected int idMeasureHistory;
diff --git a3-client/src/introsde/assignment/soap/Measure.java a3-client/src/introsde/assignment/soap/Measure.java
index e505a8f..c701f89 100644
--- a3-client/src/introsde/assignment/soap/Measure.java
+++ a3-client/src/introsde/assignment/soap/Measure.java
@@ -4,6 +4,7 @@ package introsde.assignment.soap;
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlType;
 
 
@@ -34,6 +35,7 @@ import javax.xml.bind.annotation.XmlType;
     "measureDefinition",
     "value"
 })
+@XmlRootElement
 public class Measure {
 
     protected int idMeasure;
diff --git a3-client/src/introsde/assignment/soap/MeasureDefinition.java a3-client/src/introsde/assignment/soap/MeasureDefinition.java
index 666fd4e..11f4bfe 100644
--- a3-client/src/introsde/assignment/soap/MeasureDefinition.java
+++ a3-client/src/introsde/assignment/soap/MeasureDefinition.java
@@ -6,6 +6,7 @@ import java.util.List;
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlType;
 
 
@@ -38,6 +39,7 @@ import javax.xml.bind.annotation.XmlType;
     "measureName",
     "measureType"
 })
+@XmlRootElement
 public class MeasureDefinition {
 
     protected int idMeasureDef;
